src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-FLOPS.json

```diff
@@ -0,0 +1,44 @@
+{
+  "Description": "FLOPs GPU Performance Workload",
+  "Actions": [
+    {
+      "Type": "DXFLOPSExecutor",
+      "Parameters": {
+        "ShaderIterations": 100000,
+        "Precision": "f32",
+        "PackageName": "DXMicrobenchmarks"
+      }
+    }
+  ],
+  "Monitors": [
+    {
+      "Type": "AmdSmiMonitor",
+      "Parameters": {
+        "MonitorFrequency": "00:00:05",
+        "MonitorWarmupPeriod": "00:00:10"
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "AMDGPUDriverInstallation",
+      "Parameters": {
+        "Scenario": "InstallDriver",
+        "BlobContainer": "packages",
+        "BlobName": "",

```

Comment:
Is the AMD GPU driver a public package? If the one you are using is not, we can upload some public ones and put them here, so that at least people can use it.